bin/capsule-certs-generate

```diff
@@ -31,6 +31,7 @@ CONFIG_CONTENT = <<EOF
 :answer_file: #{answers_file.path}
 :installer_dir: #{INSTALLER_DIR}
 :installer_name: 'foreman-installer'
+:product_name: 'Katello'
```

Comment:
There can be problem in `capsule-certs-generate` (https://github.com/Katello/katello-installer/blob/master/hooks/post/10-post_install.rb#L56) which is implemented using `kafo`. There the config has no scenario name and adding there scenario name 'Satellite' seems to be hackish. 

I agree about the `product_name`. For the capsule example it would make more sense to have `server_name` or `server_product_name` and `proxy_name` (`proxy_product_name`) or something similar.